chore: add e2e and docs for env file (#3159)
aws/copilot-cli: site/content/docs/developing/environment-variables.en.md

![Editing the manifest to add env vars](https://raw.githubusercontent.com/kohidave/ecs-cliv2-demos/master/env-vars-edit.svg?sanitize=true)

> Additionally, if you want to specify environment variables in bulk, you can add them in your [env file](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/taskdef-envfiles.html#taskdef-envfiles-considerations) and pass its location to your [manifest](../manifest/overview.en.md) in the `env_file` section.

Reviewer: ```suggestion
Additionally, if you want to add environment variables in bulk, you can list them in an [env file](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/taskdef-envfiles.html#taskdef-envfiles-considerations) and specify its location in the `env_file` field of your [manifest](../manifest/overview.en.md).
```

Reviewer 2: Can we also mention that the path should be from the root of the workspace?

```yaml
# in copilot/{service name}/manifest.yml
env_file: log.env
```

And in `log.env` we could have

> ```

Reviewer: super tiny nit: maybe the example contents of `log.env` should have more than one env var since we're showing how to add multiple at once?

#This is a comment and will be ignored
LOG_LEVEL=debug
```

!!! Attention

>     You can only specify one env file in each of your workspace.

Reviewer: ```suggestion
    You can specify only one env file per workspace.
```

Reviewer 2: Do we need this attention block? won't it error if they specify a list when unmarshaling the manifest?

Author: Yeah that's true. I'll remove this block.
